### ISSUE TITLE
Playground feature request: ability to import file from disk

### ISSUE THREAD

**Issue by Zireael07** [kleisauke/wasm-vips#47]
Title.

I'd love to see the result of some operations on my own pictures <3

**Comment by kleisauke** [comment]
This should already be fairly straightforward, see for example [this playground link](https://wasm-vips.kleisauke.nl/playground/?deflate=fZNBi9swEIX_ymAKccpWSQ-9tHEgKQ0sLBSW7ikOrGyPHaWyZKTxhhDy3zuSvUlzaC_B0rz53htJ2SalNZ6gVhpfOm1lBRlUtuxbNCQapB8aw-f69Filkz4qJtNvubk1CGvKvTQNcmfdm5KUNZBO4ZwbAFVDSnvlRdB7odE0tIcsy2A-BYfUO8OwoBxzOKYYPMKG9c8oK3RpsAOusNEYMMU3zjSFbDm4AMxm8BRqEmrrWkmEFahWcqja2RaKvq7RDVKNxCWmvKnOi8cgEuy4Yd06yga6IOl4fuHQ95pChqvTquv0ia0a2XuvpIFC9yM8glUrYilsp1_E_K55o4zU-vQAR6cIgfbI5xAsgCwzC22LQTscCM0Z2KGLY5mSo9rjeCLvEtvTEHywjtxfdpxlIg5dE2_sL-bnfzBvIqtRaNukr985bEh2R-UN-xs-nBn0iRNeoFVaK4_cWXnxeo-iONN4rWv-TLfXxLsHOAOdOvwKk3hds0OHzQQu94ED4OX5iRn8K0qHkvBnccCSeJ2G6r1etc3_njHbdz1dD4XVwruSO0afuH95f3XsVq38yjl5Gs_09qC3812gBG3ykCyUYS6oKsuT4a-SJ3E6Xgc5r2RZYke8jtP62cc8WS4Kt8zNIoSOrUM8LkRosvsD). Though, perhaps this could be made easier via some kind of GUI. 

Let's label this as https://github.com/kleisauke/wasm-vips/labels/enhancement. Very happy to accept a PR, if you're able.

**Comment by Zireael07** [comment]
For GUI, literally all you'd need is a <input type="file" id="fileInput" name="file" accept="image/*" /> I imagine...

**Comment by kleisauke** [comment]
Indeed, I meant some kind of "Files"-tab where you can manage the preloaded files and perhaps have them persist to IndexedDB. But maybe that's too complicated.

For now, I can recommend the playground link above.
